### PR TITLE
[IMP] account: add method to get payments

### PR DESCRIPTION
**Pull request by jco-odoo**
Description of the issue/feature this PR addresses:

Add a method to get payments on the invoice

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by jco-odoo** [comment]
@smetl Can you check this for merging?

File: addons/account/models/account_move.py [hunk 1]
```diff
@@ -1732,6 +1732,13 @@ def _get_cash_basis_matched_percentage(self):
         else:
             return abs(total_reconciled / total_amount)
 
+    def _get_invoice_payments(self):
```
Comment (smetl) [inline addons/account/models/account_move.py]: `_get_reconciled_payments(self)`?

File: addons/account/models/account_move.py [hunk 2]
```diff
@@ -1732,6 +1732,13 @@ def _get_cash_basis_matched_percentage(self):
...
+        """Used to get payments related to the invoice"""
```
Comment (smetl) [inline addons/account/models/account_move.py]: Helper used to retrieve the reconciled payments on these journal entries.

**Comment by jco-odoo** [comment]
@smetl Adapted.  Can you give it a go?

**Comment by smetl** [comment]
@robodoo r+

**Comment by smetl** [comment]
@robodoo r+

**Comment by smetl** [comment]
@robodoo retry

**Pull request closed without merging**